integration_tests/projects/001_flow_run_with_selectors/logs/dbt.log.legacy

```diff
@@ -0,0 +1,267 @@
+2022-03-09 07:05:18.387730 (MainThread): 'soft_unicode' has been renamed to 'soft_str'. The old name will be removed in MarkupSafe 2.1.
```

Comment:
Do we really want this file?

Reply:
yea looks like these didn't get gitignored sorry about it